serve.go

```diff
@@ -100,7 +100,7 @@ func (s *Server) setupDefaultListener() error {
 	}
 	addr := s.Server.Addr
 	if addr == "" {
-		addr = "127.0.0.1:9999"
+		addr = ":9999"
```

Comment:
please check TestPetstoreOpenAPIGeneration

it seems broken now

Reply:
i will check now

Reply:
"I don't see any problem, but I can add host:port by default.
![image](https://github.com/user-attachments/assets/98667548-d4e2-480b-8176-4003d67f306e)

![image](https://github.com/user-attachments/assets/94fde812-03b9-4d47-90ce-c49e8014c2f2)

![image](https://github.com/user-attachments/assets/e238a249-bfc1-4c7d-a5eb-52eb572d274d)

Reply:
I did it in https://github.com/Ignaciojeria/fuego/pull/1

But let me ask you the question here.

Why did you remove the local host? To listen on all interfaces?

Was this changes needed in this exact PR?

Reply:
Now I maintained localhost as the default to preserve the original implementation